lib/Dashboard/Model/Incidents.pm

```diff
@@ -48,7 +48,7 @@ sub build_nr ($self, $inc) {
 
 sub find ($self, $options = {}) {
   my $incidents = $self->pg->db->query(
-    'SELECT number, project, packages, rr_number, review, review_qam, approved, emu, ARRAY_AGG(c.name) as channels
+    'SELECT number, project, packages, rr_number, review, review_qam, approved, emu, crd, ARRAY_AGG(c.name) as channels
```

Comment:
And the value needs to be assigned in the code below. Like i said on Slack, it's best to search for an existing field like `emu` and replicate that code.